static/app/styles/global.tsx

```diff
@@ -193,6 +193,17 @@ const styles = (theme: Theme, isDark: boolean) => css`
             border-bottom-color: ${theme.background};
           }
         }
+        .context-summary .context-item.darwin .context-item-icon,
+        .context-summary .context-item.ios .context-item-icon,
+        .context-summary .context-item.macos .context-item-icon,
+        .context-summary .context-item.tvos .context-item-icon,
+        .context-summary .context-item.mac-os-x .context-item-icon,
+        .context-summary .context-item.mac .context-item-icon,
+        .context-summary .context-item.apple .context-item-icon,
+        .context-summary .context-item.watchos .context-item-icon {
+          filter: invert(100%);
```

Comment:
😬 

```css
.context-summary .context-item {
  ${['darwin', 'ios', 'macos', 'tvos', 'mac-os-x', 'mac', 'apple', 'watchos']
    .map(p => `&.${p} .context-item-icon`)
    .join(',')} {
    filter: invert(100%);
    opacity: 0.8;
  }
}
```

Reply:
lol tbh I could use the `:is` selector for this